component-overview/examples/dropdown/SearchableDropdown-isLoading.jsx

```diff
@@ -1,5 +1,5 @@
 import { InputGroup } from '@sb1/ffe-form-react';
-import SearchableDropdown from '@sb1/ffe-searchable-dropdown-react';
+import {SearchableDropdown} from '@sb1/ffe-searchable-dropdown-react';
```

Comment:
Samme de andre stedene :smile: 
```suggestion
import { SearchableDropdown } from '@sb1/ffe-searchable-dropdown-react';
```

Reply:
Ordnet